Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -106,6 +106,25 @@
 				<div class="neos-modal-backdrop neos-in"></div>
 			</div>
 
+			<div class="neos-hide" id="discardSelected">
+				<div class="neos-modal-centered">
+					<div class="neos-modal-content">
+						<div class="neos-modal-header">
+							<button type="button" class="neos-close neos-button" data-dismiss="modal"></button>
+							<div class="neos-header">{neos:backend.translate(id: 'workspaces.discardSelectedChangesInWorkspaceConfirmation', arguments: {0: selectedWorkspaceLabel}, source: 'Modules', package: 'Neos.Neos')}</div>
+						</div>
+						<div class="neos-modal-footer">
+							<a href="#" class="neos-button" data-dismiss="modal">{neos:backend.translate(id: 'cancel', source: 'Main', package: 'Neos.Neos')}</a>
```

Comment:
If it acts like a button it should be a `button` and not a link tag. 

Interesting: on line 94 in this file it is a button. On line 98 it's also a link tag 😲. Is this intentional?